SPECS/linux-firmware/linux-firmware.spec

```diff
@@ -19,6 +19,7 @@ Requires:       %{name}-qlogic = %{version}-%{release}
 Requires:       %{name}-qualcomm = %{version}-%{release}
 Requires:       %{name}-iwlwifi = %{version}-%{release}
 Requires:       %{name}-ice = %{version}-%{release}
+Requires:       dracut
```

Comment:
i think this should be part of qat package (line 70) 
only qat package need dracut